docs/source/release-notes/2020-2-2-Asteria.md

```diff
@@ -18,6 +17,13 @@ The numbers at the end of each item are the PR numbers in the Project Athena [re
 * Fix models not scaling correctly on load due to a low timeout; also fix resizing of models for the first time after they load. (#442)
 * Fix reloading content in serverless domains. (#447)
 
+## Domain Server
+
+The numbers at the end of each item are the PR numbers in the Project Athena [repo](https://github.com/kasenvr/project-athena/projects/2).
```

Comment:
The URL should be https://github.com/kasenvr/project-athena I believe.

Reply:
I updated the links in other release notes. I suppose it is better to let people look and figure that out as if we move from Github, then it'll be easy to just point to a new repo.